compass/ocean/tests/global_ocean/global_ocean.cfg

```diff
@@ -49,8 +49,17 @@ btr_dt_per_km = 1.5
 
 ## config options related to the initial_state step
 
+# minimum number of vertical levels, both in the open ocean and in ice-shelf
+# cavities
+min_levels = 3
+cavity_min_levels = ${min_levels}
+
+# minimum thickness of layers in ice-shelf cavities
+cavity_min_layer_thickness = 1.0
```

Comment:
Before I merge this, I'm going to determine what this actually needs to be to prevent ssh adjustment from failing.  This may take a little experimentation.